app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.it3048.mobiledeviceproject">
-
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Saw this in the professors demo video. Believe most apps will need this permisison.

Reply:
This used to be required, but not anymore.  This permission is assumed, as it is assumed that all apps will require permission to use the internet.